### ISSUE TITLE
CPU% Not Displaying in Process Manager on Windows

### ISSUE THREAD

**Issue by chgibb** [chgibb/PHAT#664]

**Comment by jsbrn** [comment]
It's been a long time since I've followed this project. Are you talking about a process manager built into PHAT?

**Comment by chgibb** [comment]
Indeed. There's some issues with it (such as this) so its still hidden behind an undocumented key combination. Its been extremely useful in development however.

**Comment by jsbrn** [comment]
How is a task manager within the scope of the project? Non-accusatory, just asking out of curiosity.

**Comment by chgibb** [comment]
Around the time it came about, we were having issues with spawned processes not exiting after signalling they were going to. Trying to figure out what exactly had not exited was extremely painful to debug with ```top```. Integrating some kind of monitor into the existing lifecycle of spawned processes seemed like a good way to debug the issue. I also thought that giving something like this the ability to kill arbitrary PHAT processes would be a good way to allow for manual and later automated failure testing, though we have not yet built that functionality. Its been extremely helpful in debugging memory leaks in specific PHAT processes as well as monitoring memory/CPU use of different parts of PHAT as it runs. This was extremely frustrating to try to do with ```top``` and ```ps``` in the past.

Most importantly, it was good fun to implement and I feel like I have better insight into the differences in cross platform PID management as a result. Though it has not yet made its debut as a documented feature, for 3 days of work, I feel it was worth it.
